_posts/2016-03-04-this-week-in-rails-turbolinks-5-rails-5-0-0-beta3-and-more.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "This week in Rails: Turbolinks 5, Rails 5.0.0.beta3 and more!"
```

Comment:
"This week in Rails: Rails 4.2.6.rc1 and 4.1.15.rc1, AS::Executor and AS::Reloader APIs and more!"